packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -145,7 +145,7 @@ class _AutomaticKeepAliveState extends State<AutomaticKeepAlive> {
       assert(() {
         if (!mounted) {
           throw FlutterError(
-            'AutomaticKeepAlive handle triggered after AutomaticKeepAlive was disposed.'
+            'AutomaticKeepAlive handle triggered after AutomaticKeepAlive was disposed. '
```

Comment:
this should actually be a newline (see docs for `FlutterError` constructor)

Reply:
Addressed in #49280